plugins/aea-cli-ipfs/aea_cli_ipfs/core.py

```diff
@@ -86,9 +91,27 @@ def add(
     """Add directory to ipfs, if not directory specified the current one will be added."""
     dir_path = dir_path or os.getcwd()
     ipfs_tool = click_context.obj
+
     click.echo(f"Starting processing: {dir_path}")
     name, hash_, _ = ipfs_tool.add(dir_path, pin=(not no_pin))
-    click.echo(f"Added: `{name}`, hash is {hash_}")
+
+    yaml_files = glob(str(Path(dir_path) / "*.yaml"))
+    if len(yaml_files) > 0:
+        (config_file_path,) = yaml_files
+        package_path, config_file = os.path.split(config_file_path)
+        item_type, _ = os.path.splitext(config_file)
+        package_path = Path(package_path)
```

Comment:
we have proper methods for loading the agent/component config files

Reply:
I checked all the methods, They require both package path and item type to load config. Since we're not passing any argument for item type in `aea ipfs add` command I need to use this logic to find out item type for the package.